### PR TITLE
[RHACS] ROX-17368: Updating Release notes for RHACS 4.0.2 patch

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.0

Issue:
https://issues.redhat.com/browse/ROX-17368

Link to docs preview:
https://60661--docspreview.netlify.app/openshift-acs/latest/release_notes/40-release-notes.html#resolved-in-version-402

SME review:
- [x] SME has approved this change.

Additional information:
Merge into `rhacs-docs` and cherry-pick into:
- `rhacs-docs-4.0`

### CODE REVIEW

**Review by charmik-redhat — APPROVED** [review]

File: release_notes/40-release-notes.adoc [hunk 1]
```diff
@@ -455,6 +456,21 @@ Release date: 18 May 2023
...
+
+Release date: 31 May 2023
+
+//bugs are not linked because ACS Jira is not public
+* This release of {product-title-short} includes a fix for:
+
+** link:https://access.redhat.com/security/cve/cve-2023-24540[CVE-2023-24540], by building {product-title-short} with updated Golang.
+//ROX-16989
+** link:https://access.redhat.com/security/cve/CVE-2023-29400[CVE-2023-29400], which fixes a vulnerability in Golang that affects templates with actions in unquoted HTML attributes.
+** link:https://access.redhat.com/security/cve/CVE-2023-24539[CVE-2023-24539], which fixes a vulnerability in Golang related to the handling of angle brackets in Cascading Style Sheets (CSS) contexts.
```
[the remark this answers is not in the document]
Comment (snarayan-redhat) [reply release_notes/40-release-notes.adoc:470]: You may also want to include `(<>)` after angle brackets

File: release_notes/40-release-notes.adoc [hunk 2]
```diff
@@ -455,6 +456,21 @@ Release date: 18 May 2023
...
+
+//ROX-17189
+* Fixed an issue that caused frequent Central pod restarts with the `context deadline exceeded` error.
```
Comment (snarayan-redhat) [inline release_notes/40-release-notes.adoc:473]: Does `context deadline exceeded` have `C` in caps? If not, this is good.

File: release_notes/40-release-notes.adoc (hunk 1 above)
Comment (snarayan-redhat) [inline release_notes/40-release-notes.adoc:470]: A small suggestion to follow parallelism. Up to you though.
```suggestion
* This release of {product-title-short} fixes following security vulnerabilities:

** link:https://access.redhat.com/security/cve/cve-2023-24540[CVE-2023-24540]: Fixed by building {product-title-short} with updated Golang.
//ROX-16989
** link:https://access.redhat.com/security/cve/CVE-2023-29400[CVE-2023-29400]: Fixed a vulnerability in Golang that affects templates with actions in unquoted HTML attributes.
** link:https://access.redhat.com/security/cve/CVE-2023-24539[CVE-2023-24539]: Fixed a vulnerability in Golang related to the handling of angle brackets in Cascading Style Sheets (CSS) contexts.
```

File: release_notes/40-release-notes.adoc [hunk 3]
```diff
@@ -455,6 +456,21 @@ Release date: 18 May 2023
...
 
+[id="resolved-in-version-402"]
+=== Resolved in version 4.0.2
+
+Release date: 31 May 2023
+
+//bugs are not linked because ACS Jira is not public
+* This release of {product-title-short} includes a fix for:
+
+** link:https://access.redhat.com/security/cve/cve-2023-24540[CVE-2023-24540], by building {product-title-short} with updated Golang.
```
Comment (snarayan-redhat) [inline release_notes/40-release-notes.adoc:467]: This bullet talks about how it was fixed and other 2 talk about what is fixed. May be you can keep it consistent to tell either what the fix is or what is fixed. I would choose latter.

**Review by snarayan-redhat — COMMENTED** [review]
Few suggestions. Otherwise looks good!

File: release_notes/40-release-notes.adoc (hunk 2 above)
Comment (agantony) [reply release_notes/40-release-notes.adoc:473]: "c" is in lower case

**Comment by kcarmichael08** [comment]
lgtm to merge!

**Pull request merged**